pandevice/base.py

```diff
@@ -2285,11 +2311,12 @@ class ParamPath(object):
         exclude (bool): Exclude this param from the resultant XML.
 
     """
-    def __init__(self, param, path=None, vartype=None,
+    def __init__(self, param, path=None, vartype=None, exist_tag=None,
```

Comment:
Coded like that, the variable name becomes part of the xpath, which doesn't allow the xpath to change between versions.